pkg/proxy/request_test.go

```diff
@@ -210,14 +210,12 @@ func TestEnvVarFile(t *testing.T) {
 }
 
 func TestInit(t *testing.T) {
-	expectedDP := int32(2)
 	ltType := apisLoadTestV1.LoadTestTypeJMeter
 	for _, ti := range []struct {
-		tag              string
-		requestFile      map[string]string
-		distributedPods  string
-		expectedResponse apisLoadTestV1.LoadTestSpec
-		expectError      bool
+		tag             string
+		requestFile     map[string]string
```

Comment:
Please add the tests with wrong file formats for all testFile, testData and envVars

Reply:
Done